RespiratoryAssistDevices/R4/resources/Questionnaire-R4-RespiratoryAssistDevices.json

```diff
@@ -483,125 +235,22 @@
         },
         {
           "linkId": "5.7",
-          "type": "group",
+          "type": "open-choice",
           "text": "Supply Order",
-          "item": [
-            {
-              "linkId": "5.7.1",
-              "text": "Note:  Items dispensed on a periodic basis — repeat multiple times",
-              "type": "display"
-            },
-            {
-              "linkId": "5.7.2",
-              "type": "group",
-              "text": "List",
-              "repeats": true,
-              "extension": [
-                {
-                  "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
-                  "valueCodeableConcept": {
-                    "coding": [
-                      {
-                        "system": "http://hl7.org/fhir/questionnaire-item-control",
-                        "code": "gtable"
-                      }
-                    ]
-                  }
-                }
-              ],
-              "item": [
-                {
-                  "linkId": "5.7.2.1",
-                  "text": "Item Description",
-                  "type": "string"
-                },
-                {
-                  "linkId": "5.7.2.2",
-                  "text": "Frequency",
-                  "type": "string"
-                },
-                {
-                  "linkId": "5.7.2.3",
-                  "text": "Duration",
-                  "type": "string"
-                },
-                {
-                  "linkId": "5.7.2.4",
-                  "text": "Quantity",
-                  "type": "string"
-                },
-                {
-                  "linkId": "5.7.2.5",
-                  "text": "Refills",
-                  "type": "string"
-                }
-              ]
-            },
-            {
-              "linkId": "5.7.3",
-              "text": "Other",
-              "type": "text"
-            }
-          ]
+          "repeats": true,
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.129"
```

Comment:
why do we change the table to a drop list?

Reply:
The supply list is defined as a valueset and we are going to use it as a drop down. We were also talking about merging the main request and supply, but different story is created to pre-populate it.

Reply:
We are missing several columns of information, frequency, duration, quantity, refills. The dropdown list covers the first column only.